Angular/src/app/model/capability.ts

```diff
@@ -0,0 +1,4 @@
+export class Capability {
```

Comment:
I think you can just make this an `interface`:

```
export interface Capability {
     departmentId: number;
     name: string;
}
```

See https://stackoverflow.com/a/51716899